src/xrCore/xrDebug.cpp

```diff
@@ -939,6 +939,6 @@ void xrDebug::Initialize(pcstr commandLine)
 #ifdef DEBUG
     ShowErrorMessage = true;
 #else
-    ShowErrorMessage = commandLine ? strstr(commandLine, "-show_error_window") : false;
+    ShowErrorMessage = commandLine ? (bool)strstr(commandLine, "-show_error_window") : false;
```

Comment:
It's better like that:
```suggestion
    ShowErrorMessage = commandLine ? !!strstr(commandLine, "-show_error_window") : false;
```